ext/bcmath/tests/bcpowmod_negative_exponent.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+bc_raisemod's expo can't be negative
+--CREDITS--
+Gabriel Caruso (carusogabriel34@gmail.com)
```

Comment:
Ok, wait, what my name is doing here?

Reply:
Oops, too late, sorry :S You can remove it from master though :)